app/services/data_migrations/delete_work_history_audits.rb

```diff
@@ -0,0 +1,24 @@
+module DataMigrations
+  class DeleteWorkHistoryAudits
+    TIMESTAMP = 20240917154444
+    MANUAL_RUN = true
+    BATCH_SIZE = 1500
```

Comment:
We could run this automatically and potentially up the Batch size to 5000 - we did 10000 in the big delete #10002